src/pages/basic-concepts/content-markdown.md

```diff
@@ -3,7 +3,143 @@ layout: ~/layouts/Main.astro
 title: Content (Markdown)
 ---
 
-- Layouts
-- MDX
+Astro comes with out-of-the-box Markdown support powered by the expansive [remark](https://remark.js.org/) ecosystem.
 
-## Remote data
+### Remark Plugins
+
+**This is the first draft of Markdown support!** While we plan to support user-provided `remark` plugins soon, our hope is that you won't need `remark` plugins at all!
+
+In addition to [custom components inside the `<Markdown>` component](https://github.com/snowpackjs/astro/blob/main/docs/markdown.md#markdown-component), Astro comes with [Github-flavored Markdown](https://github.github.com/gfm/) support, [Footnotes](https://github.com/remarkjs/remark-footnotes) syntax, [Smartypants](https://github.com/silvenon/remark-smartypants), and syntax highlighting via [Prism](https://prismjs.com/) pre-enabled. These features are likely to be configurable in the future.
+
+### Markdown Pages
+
+Astro treats any `.md` files inside of the `/src/pages` directory as pages. These pages are processed as plain Markdown files and do not support components. If you're looking to embed rich components in your Markdown, take a look at the [Markdown Component](#markdown-component) section.
+
+`layout`
+
+The only special Frontmatter key is `layout`, which defines the relative path to a `.astro` component which should wrap your Markdown content.
+
+`src/pages/index.md`
+
+```jsx
+---
+layout: ../layouts/main.astro
+---
+
+# Hello World!
+```
+
+Layout files are normal `.astro` components. Any Frontmatter defined in your `.md` page will be exposed to the Layout component as the `content` prop. `content` also has an `astro` key which holds special metadata about your file, like the complete Markdown `source` and a `headings` object.
+
+The rendered Markdown content is placed into the default `<Slot />` element.
+
+`src/layouts/main.astro`
+
+```jsx
+---
+export let content;
+---
+
+<html>
+  <head>
+    <title>{content.title}</title>
+  </head>
+
+  <body>
+    <slot/>
```

Comment:
is it `<Slot>` or `<slot>` ? 
There is a conflict in terms between the two on line 42 and ln 49,